app/build.gradle

```diff
@@ -1,16 +1,34 @@
+buildscript {
+  repositories {
+    jcenter()
+    maven { url 'https://maven.fabric.io/public' }
+  }
+
+  dependencies {
+    // The Fabric Gradle plugin uses an open ended version to react
+    // quickly to Android tooling updates
+    classpath 'io.fabric.tools:gradle:1.+'
```

Comment:
avoid 1.+ version, use specific versions, because a gradle sync can broke the project